src/tls.c

```diff
@@ -650,12 +649,46 @@ static void updateSSLEvent(tls_connection *conn) {
     if (!need_write && (mask & AE_WRITABLE)) aeDeleteFileEvent(server.el, conn->c.fd, AE_WRITABLE);
 }
 
+static int TLSHandleAcceptResult(tls_connection *conn, int call_handler_on_error) {
```

Comment:
can we assert here if we are NOT in CONN_STATE_ACCEPTING ?

Reply:
You mean we are IN CONN_STATE_ACCEPTING, right?